setup.py

```diff
@@ -107,7 +107,7 @@ def run_tests(self):
         'Topic :: Office/Business :: Financial :: Point-Of-Sale',
     ],
     install_requires=[
-        'pyusb>=1.0.0',
+        'libusb1',
```

Comment:
Are all versions of this lib OK?

Reply:
I haven't checked other versions of libusb1.